cmd/erasure-server-pool-decom.go

```diff
@@ -1077,11 +1056,15 @@ func (z *erasureServerPools) Decommission(ctx context.Context, idx int) error {
 	}
 
 	// Make pool unwritable before decommissioning.
-	if err := z.StartDecommission(ctx, idx); err != nil {
+	if err := z.StartDecommission(ctx, indices...); err != nil {
 		return err
 	}
 
-	go z.doDecommissionInRoutine(ctx, idx)
+	go func() {
```

Comment:
Didn't we agree that decommissinong won't be parallel ?

Reply:
I *think* the intent is that `z.poolMetaMutex.Lock();	defer z.poolMetaMutex.Unlock()` is making it serial below, but I would like to have that confirmed as well. Had the same Q.

Reply:
Decommission is for one pool at a time @vadmeste